### ISSUE TITLE
NoClassDefFoundError for LandsHook

### ISSUE THREAD

**Issue by SlimeDog** [XZot1K/PhysicsToGo#9]
Spigot 1.15.2
PTG 2.0.2

Logged error on start-up
```
[14:59:06] [Server thread/INFO]: [PhysicsToGo] Enabling PhysicsToGo v2.0.2
[14:59:06] [Server thread/INFO]: [PhysicsToGo] Everything inside the configuration seems to be up to date. (Took 2ms)
[14:59:06] [Server thread/ERROR]: Error occurred while enabling PhysicsToGo v2.0.2 (Is it up to date?)
java.lang.NoClassDefFoundError: me/angeschossen/lands/api/integration/LandsIntegration
        at xzot1k.plugins.ptg.core.hooks.LandsHook.<init>(LandsHook.java:15) ~[?:?]
        at xzot1k.plugins.ptg.PhysicsToGo.onEnable(PhysicsToGo.java:86) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:457) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:274) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: me.angeschossen.lands.api.integration.LandsIntegration
        at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:96) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        ... 11 more
```

**Comment by XZot1K** [comment]
> Spigot 1.15.2
> PTG 2.0.2
> 
> Logged error on start-up
> 
> ```
> [14:59:06] [Server thread/INFO]: [PhysicsToGo] Enabling PhysicsToGo v2.0.2
> [14:59:06] [Server thread/INFO]: [PhysicsToGo] Everything inside the configuration seems to be up to date. (Took 2ms)
> [14:59:06] [Server thread/ERROR]: Error occurred while enabling PhysicsToGo v2.0.2 (Is it up to date?)
> java.lang.NoClassDefFoundError: me/angeschossen/lands/api/integration/LandsIntegration
>         at xzot1k.plugins.ptg.core.hooks.LandsHook.<init>(LandsHook.java:15) ~[?:?]
>         at xzot1k.plugins.ptg.PhysicsToGo.onEnable(PhysicsToGo.java:86) ~[?:?]
>         at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:457) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:274) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784) [spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at java.lang.Thread.run(Thread.java:834) [?:?]
> Caused by: java.lang.ClassNotFoundException: me.angeschossen.lands.api.integration.LandsIntegration
>         at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
>         at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:96) ~[spigot-1.15.2.jar-2020-03-31-1855:git-Spigot-a03b1fd-bbe3d58]
>         at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
>         at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
>         ... 11 more
> ```

Thank you for the report! Got that fixed. :)

**Comment by SlimeDog** [comment]
Confirmed fix in 2.0.3
